calculator/Calculator.java

```diff
@@ -0,0 +1,56 @@
+package calculator;
+
+import java.util.*;
+public class Calculator {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		Scanner sc=new Scanner(System.in);
+		System.out.println("Enter first number:");
+		double a=sc.nextDouble();

```

Comment:
What happens if user gives string as input ? Handle input validation here, if exception is thrown, handle it.